src/main/resources/application-dev-gcp.properties

```diff
@@ -10,3 +10,6 @@ hentinntektlistebolk.scopes=api://dev-fss.team-inntekt.inntektskomponenten/.defa
 organisasjon.rs.url=https://ereg-services.dev-fss-pub.nais.io/api/v2/organisasjon
 
 altinn.scopes=api://dev-gcp.arbeidsgiver.altinn-rettigheter-proxy/.default
+
+dokarkiv.base.url=http://dokarkiv.teamdokumenthandtering/rest/journalpostapi/v1/journalpost
```

Comment:
base.url kunne egentlig flyttes opp til application.properties siden den er lik for dev og prod.

Reply:
https://dokarkiv-q2.dev-fss-pub.nais.io bør brukes

Reply:
Lagt inn!